.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   # latest LTS
   - SONAR_VERSION=6.7.6 SONAR_JAVA_VERSION=5.2.0.13398
   # latest releases
-  - SONAR_VERSION=7.5 SONAR_JAVA_VERSION=5.9.2.16552
+  - SONAR_VERSION=7.6 SONAR_JAVA_VERSION=5.12.1.17771
```

Comment:
As documented in README, in this branch we support only SonarQube `6.7.1~7.5`.
Then could you revert the `SONAR_VERSION` to `7.5`?